tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>Would do something like this:</p><p> </p><p>1. "Train a simple sequential model with MNIST dataset from scratch". -&gt;  "Train a&nbsp;<code><a href="https://www.tensorflow.org/api_docs/python/tf/keras" target="_blank">tf.keras</a></code>&nbsp;model for MNIST from scratch."</p><p> </p><p>Reason: conciseness. It is a sequential model, but that's not important for getting the story across. tf.keras is important to mention since some may utilize other implementations of Keras, which the tool is not compatible with.</p><p> </p><p>2. -&gt; Fine tune the model by applying the weight clustering API and see the accuracy.</p><p> </p><p>3. Create Nx smaller TF and TFLite models from weight clustering.</p><p> </p><p>Reason: highlight clustering standalone and that this works for both TF and TFLite.</p><p> </p><p>4. Create a Nx smaller TFLite model from combining weight clustering and quantization.</p><p> </p><p>Reason: clarify 11x is from combining the two.</p><p> </p><p>5. See the persistence of accuracy from TF to TFLite.</p><p> </p><p>Reason: removed "see the latency benefits on mobile, try out the TFLite examples in the&nbsp;<a href="https://www.tensorflow.org/lite/models" target="_blank">TFLite app repository</a>" since that's specific to quantization. </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>